src/Adapters/ADTAdapter.ts

```diff
@@ -473,6 +475,30 @@ export default class ADTAdapter implements IADTAdapter {
         );
     }
 
+    advancedSearchADTTwins(
+        params: AdapterMethodParamsForAdvancedSearchADTwins
+    ) {
+        const adapterMethodSandbox = new AdapterMethodSandbox(this.authService);
+        console.log(params);
+        return adapterMethodSandbox.safelyFetchDataCancellableAxiosPromise(
+            ADTAdapterAdvancedSearchData,
+            {
+                method: 'post',
+                url: `${this.adtProxyServerPath}/query`,
+                headers: {
+                    'x-adt-host': this.adtHostUrl
+                },
+                params: {
+                    'api-version': ADT_ApiVersion
+                },
+                data: {
+                    query: params.query,
+                    continuationToken: params.continuationToken
```

Comment:
are you going to do anything with the continuation token? Or does the caller have to deal with that?

Reply:
There is a TODO item for this, in case the response yields paginated results have the details list scroll trigger a search with continuation token to get the remaining items.

Reply:
This is actually complex, we don't want to stream 100K twins to the client.  I think for now it's perhaps best to stop at, say, 1000, and show content in the results that says "First 1000, of many" and an info bubble that tells them to filter more.  This is very very similar to what we did for this problem in TSI